WcaOnRails/spec/jobs/sync_mailing_lists_job_spec.rb

```diff
@@ -8,15 +8,38 @@
+    europe_north_baltic_states_delegate = FactoryBot.create :delegate
+    europe_north_baltic_states_delegate.senior_delegate.update(region: "Europe North & Baltic States (Denmark)")
+    europe_west_delegate = FactoryBot.create :delegate
+    europe_west_delegate.senior_delegate.update(region: "Europe West (Germany)")
+    latin_america_delegate = FactoryBot.create :delegate
+    latin_america_delegate.senior_delegate.update(region: "Latin America (Peru)")
+    oceania_delegate = FactoryBot.create :delegate
+    oceania_delegate.senior_delegate.update(region: "Oceania (Australia, Victoria)")
+    usa_canada_delegate = FactoryBot.create :delegate
+    usa_canada_delegate.senior_delegate.update(region: "USA & Canada (USA, Oregon)")
+
     expect(GsuiteMailingLists).to receive(:sync_group).with(
       "delegates@worldcubeassociation.org",
-      a_collection_containing_exactly(candidate_delegate.email, candidate_delegate.senior_delegate.email, delegate.email, delegate.senior_delegate.email, senior_delegate.email),
+      a_collection_containing_exactly(candidate_delegate.email, candidate_delegate.senior_delegate.email, delegate.email, delegate.senior_delegate.email, senior_delegate.email, africa_delegate.email, africa_delegate.senior_delegate.email, asia_east_delegate.email, asia_east_delegate.senior_delegate.email, asia_japan_delegate.email, asia_japan_delegate.senior_delegate.email, asia_southeast_delegate.email, asia_southeast_delegate.senior_delegate.email, asia_west_india_delegate.email, asia_west_india_delegate.senior_delegate.email, europe_east_middle_east_delegate.email, europe_east_middle_east_delegate.senior_delegate.email, europe_north_baltic_states_delegate.email, europe_north_baltic_states_delegate.senior_delegate.email, europe_west_delegate.email, europe_west_delegate.senior_delegate.email, latin_america_delegate.email, latin_america_delegate.senior_delegate.email, oceania_delegate.email, oceania_delegate.senior_delegate.email, usa_canada_delegate.email, usa_canada_delegate.senior_delegate.email),
     )
 
     # seniors@ mailing list
     expect(GsuiteMailingLists).to receive(:sync_group).with(
       "seniors@worldcubeassociation.org",
-      a_collection_containing_exactly(candidate_delegate.senior_delegate.email, delegate.senior_delegate.email, senior_delegate.email),
+      a_collection_containing_exactly(candidate_delegate.senior_delegate.email, delegate.senior_delegate.email, senior_delegate.email, africa_delegate.senior_delegate.email, asia_east_delegate.senior_delegate.email, asia_japan_delegate.senior_delegate.email, asia_southeast_delegate.senior_delegate.email, asia_west_india_delegate.senior_delegate.email, europe_east_middle_east_delegate.senior_delegate.email, europe_north_baltic_states_delegate.senior_delegate.email, europe_west_delegate.senior_delegate.email, latin_america_delegate.senior_delegate.email, oceania_delegate.senior_delegate.email, usa_canada_delegate.senior_delegate.email),
```

Comment:
Could you split this (and the other long line) up into multiple lines? Let's try not to go over 100 characters.

Reply:
Yep, on it!